devimint/src/util.rs

```diff
@@ -209,6 +209,13 @@ impl Command {
         self
     }
 
+    pub fn args<T: ToString>(mut self, args: impl IntoIterator<Item = T>) -> Self {
```

Comment:
Kept it from a different approach. It's useful if you want to pass an array of args.